client/sites-dashboard/components/sites-site-plan.tsx

```diff
@@ -32,7 +30,7 @@ interface SitePlanProps {
 const STAGING_PLAN_LABEL = 'Staging';
 
 export const SitePlan = ( { site, userId }: SitePlanProps ) => {
-	const isWpcomStagingSite = useSelector( ( state ) => isSiteWpcomStaging( state, site.ID ) );
+	const isWpcomStagingSite = site?.is_wpcom_staging_site ?? false;
```

Comment:
> I'm guessing this was changed to make it easier to write the unit tests? I think it's a good change because why get the data from the Redux store when we've already been passed all the state we need via props.

@p-jackson Yes to both.